logstash-core-event-java/src/main/java/com/logstash/Javafier.java

```diff
@@ -200,5 +226,9 @@ public static Object deep(IRubyObject o) {
         ArrayJavaProxy,
         ConcreteJavaProxy
     }
+
+    private static String missingHandlerString(String fmt, String... subs) {
```

Comment:
not sure why this method is needed...? you can keep it if you want but if I ever modify this area I might inadvertently replace it with a direct `String.format` call 👼